app/views/layouts/application.html.slim

```diff
@@ -1,12 +1,11 @@
-- content_for :template, yield(:title) || t(:loans)
-
 - content_for :head
   = javascript_include_tag "application"
   = stylesheet_link_tag "application", :media => "all"
-  = csrf_meta_tags
+  // = csrf_meta_tags
+
+- content_for :template, yield(:title)
 
 - content_for :content
-  = render 'alerts'
```

Comment:
Ah yes, and looks like Mel already fixed this in her branch. 

I didn't expect anyone else to be working so late :)